### DISCUSSION TITLE
my init.lua lives in another folder

### DISCUSSION THREAD

**Discussion by juantascon** [folke/lua-dev.nvim#137] [Q&A]
my init.lua lives in another folder that I symlink to .config/nvim/init.lua
How can I enable neodev in that folder too?

**Comment by folke** [comment]
check the docs. It's literally in there

**Comment by AugustaFleisneru** [comment]
You can enable neodev in your symlinked folder by creating a neodev.toml file in that folder and configuring it according to your preferences. This file should be in the same format as the neodev.toml file in your ~/.config/nvim folder. You may need to update the path to your neovim executable in the neodev.toml file to ensure that neodev can find and run your neovim instance.
